README.md

```
@@ -2,313 +2,1521 @@
+- [Stefan Bohacek](https://fourtonfish.com)
+- [Stefan Topalovic](https://www.stefantopalovic.com)
+- [Stephanie Lin](https://lin-stephanie.github.io)
+- [Stéphane Chan Hiou Kong](https://www.chan-stephane.me) ([@Chan-Stephane](https://github.com/chan-stephane)) [Software Engineer]
+- [Subhan Nadeem](https://subhan-dev-portfolio.vercel.app)
+  ([@Satishjhanwer](https://github.com/satishjhanwer))
+- [Subhasish Das](https://subhasish-portfolio.vercel.app)
+- [Sudarshan Rai](https://sudarshanrai.com.np) [Full Stack Developer]
+- [Sudev Thapa Magar](https://www.sudevmagar.tech)
+- [Sujal Bedre](https://sujal-bedre.vercel.app) [Ai/Ml Dev && Mern-Stack Dev]
+- [Sumeet Haldar](https://hsumeet.site)
+- [Sumeet](https://full-stack-liard-eight.vercel.app)[Software Developer]
```

markdownlint-cli2 (0.18.1)

Reversed link syntax
(https://full-stack-liard-eight.vercel.app)